src/rcj_flask.py

```diff
@@ -94,3 +94,72 @@ def get_runs():
 def get_runs_competition(competition):
     return jsonify({'runs': g.rcj.get_runs_competition(competition)})
 
+def api_v2_check_auth(request):
+    username = request['referee']['name']
+    password = request['referee']['auth']
```

Comment:
Check if `'referee'` with the attributes `'name'` and `'auth'` exists before accessing them.

Reply:
https://github.com/rcjberlin/rcj-server/blob/e41d2bed48182bd54b1cc712eea0f929a2f9f6d7/src/rcj_flask.py#L94-L100